src/structures/queue.js

```diff
@@ -0,0 +1,21 @@
+class MyQueue{
+    elements = [];
+    
+    push(element){ 
+        this.elements.push(element);
+
+    }
+    
+    shift(){
+        return this.elements.shift();
+    }
+}
+
+const QUEUE = new MyQueue();
+QUEUE.push(1);
+QUEUE.push(2);
+QUEUE.push(3);
+QUEUE.shift();
+
+
+export default MyQueue;
```

Comment:
Prefer named export, refer: https://dev.to/phuocng/avoid-using-default-exports-a1c